doc/adrs/022_add_support_for_selection_dialog.adoc

```diff
@@ -0,0 +1,129 @@
+= ADR-022 - Add support for selection dialog
+
+== Context
+
+We want to support Sirius RCP's `Select Model Element Variable` for the tools of the diagram.
+
+== Decision
+
+We will only support one and only one `Select Model Element Variable` per tool and we will start with the Node Creation Tool for the moment.
+
```

Comment:
The `Select Model Element Variable` will only support single selection and list display (no tree display) for the moment.

Reply:
Added to the document